pages/1.7/release-notes/1.7.3/index.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for 1.7.3
+title: Release Notes for 1.7.3
+menuWeight: 2
+excerpt:
+
+enterprise: false
+---
+
+The release notes provide a list of useful topics and links for DC/OS.
+
+# <a name="known-issues"></a>Known Issues and Limitations
```

Comment:
We should either list all the new features in 1.7 or check this list against the fixes in 1.7.2 and 3, and note that these are known issues that persist in 1.7.3.